readme.md

```diff
@@ -22,7 +22,115 @@ The product is an AI-based mental health app with the main feature being a highl
 
 ## 4. Database Schema
 
-TODO: add database schema
+```mermaid
+
+erDiagram
+   users {
+      int id PK
+      dateTime created_at
+      dateTime updated_at
+      citext email "unique constraint"
+      text password_hash
+      text password_salt
+      int role_id FK
+   }
+   user_roles {
+      int id PK
+      dateTime created_at
+      dateTime updated_at
+      varchar name
+      varchar key "chatbot and user"
+   }
+   user_details {
+      int id PK
+      dateTime created_at
+      dateTime updated_at
+      int user_id FK
+      varchar full_name
+      int avatar_id FK "may be null if user has no avatar"
+      text survey
+   }
+   files {
+      int id PK
+      dateTime created_at
+      dateTime update_at
```

Comment:
```suggestion
      dateTime updated_at
```
Same for user_preferences, chat_topics, chats, chat_messages, meditation_topics, meditation_entries, journal_topics and journal_entries